src/commands.ts

```diff
@@ -13,8 +13,16 @@ import { type MaybeArray } from "./utils.ts";
 
 export type SetMyCommandsParams = {
     /**
-     * Scope
+     * Interface for grouping {@link BotCommand}'s that might or might not be related to each other
+     * by scope and/or language.
+     *
+     * If @param scope it's defined, they will only take effect on that scope.
+     * If @param language_code it's defined, they will only take effect on users of that language.
+     * 'Takes effect' in this case means: being displayed in the user commands menu.
```

Comment:
This should go before the `export type` line, because it refers to the whole type